packages/core/utils/src/generateBuildMetrics.js

```diff
@@ -85,10 +86,16 @@ async function getSourcemapSizes(
     }
 
     let sizeMap = new Map();
+    for (let source of sources) {
+      sizeMap.set(source, 0);
+    }
+
     for (let i = 0; i < sourceSizes.length; i++) {
       sizeMap.set(sources[i], sourceSizes[i]);
     }
 
+    sizeMap.set('unknown-origin', unknownOrigin);
```

Comment:
Maybe do `sizeMap.set(null, unknownOrigin)` instead so that the reporter can do some special formatting for this pseudo asset? Right now it looks like a regular asset that is just called "unknown-origin".

Reply:
Sure, I'm just not sure how I want to display it.